packages/core/src/bar-chart.ts

```diff
@@ -100,7 +100,8 @@ export class BarChart extends BaseAxisChart {
 		if (xScale) {
 			this.x = xScale;
 		} else {
-			this.x = scaleBand().rangeRound([0, width]).padding(Configuration.bars.spacing.datasets);
+			this.x = this.options.rtl ?  scaleBand().rangeRound([width, 0]) : scaleBand().rangeRound([0, width]);
```

Comment:
Maybe a better way to do this?
Maybe take out the `rangeRound(...)` and store in a variable, and if `this.options.rtl`, then we can `reverse()` the array